SU-SOE/soe_basic
# D8CORE-2443: Fix to font size

# READY FOR REVIEW

# Summary
- Fix to the CTA font size

# Needed By (Date)
- Friday

# Urgency
- Med

# Steps to Test

1. Pull in this branch
2. Build
3. Add a CTA list. Verify it matches the designs in the ticket.
https://stanfordits.atlassian.net/browse/D8CORE-2443

# Affected Projects or Products
- SOE Subtheme

# Associated Issues and/or People
- D8CORE-2443

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
